src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -14,6 +14,7 @@
 #include "ofi_impl.h"
 #include <opa_primitives.h>
 
+#define MAX_SHMKEY_LEN 64
```

Comment:
Can you namespace any defines to avoid possible collisions. One of the drawbacks of the header-only implementation...